rulesets/src/collections.ruleset.yml

```
@@ -0,0 +1,136 @@
+rules:
+  ##### General #####
+  sps-no-collection-paging-capability:
+    description: Response bodies from collection endpoints must offer paging capability.
```

Check other descriptions for consistency in the MUST vs SHOULD usage:
```suggestion
    description: Response bodies from collection endpoints MUST offer paging capability.
```